code/game/gamemodes/ninja/ninja.dm

```diff
@@ -1,16 +1,16 @@
 /datum/game_mode/ninja
-	name = "ninja"
+	name = "ninjas"
 	round_description = "An agent of the Spider Clan is onboard the station!"
 	extended_round_description = "What was that?! Was that a person or did your eyes just play tricks on you? \
-		You have no idea. That slim-suited, cryptic individual is an enigma to you and all of your knowledge. \
+		You have no idea. Those slim-suited, cryptic individuals are an enigma to you and all of your knowledge. \
 		Their purpose is unknown. Their mission is unknown. How they arrived to this secure and isolated \
-		section of the galaxy, you don't know. What you do know is that there is a silent shadow-stalker piercing \
+		section of the galaxy, you don't know. What you do know is that there are silent shadow-stalkers piercing \
 		through the defenses of the station with technological capabilities eons ahead of your time. They can avoid \
 		the omniscience of the AI and rival the most hardened weapons your station is capable of. Tread lightly and \
-		only hope this unknown assassin isn't here for you."
+		only hope these unknown assassins isn't here for you."
 	config_tag = "ninja"
 	required_players = 2
```

Comment:
This needs to change to 15-20 because this var is how many ready players you need for round to start. Number is debatable.